crates/tabby/src/services/completion/completion_prompt.rs

```diff
@@ -173,7 +173,7 @@ lazy_static! {
 }
 
 fn tokenize_text(text: &str) -> Vec<String> {
-    TOKENIZER.split(text).map(|x| x.to_owned()).collect()
+    TOKENIZER.split(text).map(|x| x.to_owned()).filter(|x| !x.is_empty()).collect()
```

Comment:
Could you please add a unit test case within the mod test module to cover the differences resulting from the addition of this filter?